aws-greengrass-testing-platform/aws-greengrass-testing-platform-api/src/main/java/com/aws/greengrass/testing/platform/PlatformResolver.java

```diff
@@ -13,22 +13,26 @@
 import com.aws.greengrass.testing.platform.macos.MacosPlatform;
 import com.aws.greengrass.testing.platform.windows.WindowsPlatform;
 import com.google.common.annotations.VisibleForTesting;
+import io.cucumber.guice.ScenarioScoped;
 
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import javax.inject.Inject;
 
-public class PlatformResolver {
+@ScenarioScoped
+public class PlatformResolver implements Platform {
```

Comment:
Yoyu don't need to change the PlatformResolver

Reply:
resolved